### PR TITLE
Add CJIS Mapping

### PR DESCRIPTION
**Pull request by redhatrises**
- Fix some stigid typos as well

### CODE REVIEW

**Comment by redhatrises** [comment]
@shawndwells do you want to review? This is just mapping based on what @rprice submitted for the CJIS profile.

**Comment by mpreisler** [comment]
Fixed conflicts using the new github web UI. Let's see if Jenkins passes.

File: shared/transforms/shared_constants.xslt [hunk 1]
```diff
@@ -9,6 +9,7 @@
 <!-- abbreviated as references in the XCCDF-->
 <xsl:variable name="nist800-53uri">http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-53r4.pdf</xsl:variable>
 <xsl:variable name="nist800-171uri">http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-171.pdf</xsl:variable>
+<xsl:variable name="cjisd-its-uri">https://www.fbi.gov/file-respository/cjis-security-policy-v5_5_20160601-2-1.pdf</xsl:variable>
```
Comment (shawndwells) [inline shared/transforms/shared_constants.xslt]: 404 on that URL. Looks like the official URL actually has a typo in it:

https://www.fbi.gov/file-repository/cjis-security-policy-v5_5_20160601-2-1.pdf  -- in patch 
https://www.fbi.gov/file-respository/cjis-security-policy-v5_5_20160601-2-1.pdf -- official url

repository vs respository

Use https://www.fbi.gov/services/cjis/cjis-security-policy-resource-center instead?

File: shared/transforms/shared_xccdf2table-byref.xslt [hunk 2]
```diff
@@ -95,6 +95,15 @@
...
 
+                        <xsl:if test="$ref='cjis'">
+                                <xsl:for-each select="//cdf:reference[@href=$cjisd-its-uri]" >
+                                        <xsl:call-template name="rule-output">
+                                                <xsl:with-param name="refinfo" select="." />
+                                        </xsl:call-template>
+                                </xsl:for-each>
+                        </xsl:if>
+
+
```
Comment (shawndwells) [inline shared/transforms/shared_xccdf2table-byref.xslt]: extra line

File: shared/xccdf/system/accounts/pam.xml [hunk 3]
```diff
@@ -763,7 +763,7 @@ Preventing re-use of previous passwords helps ensure that a compromised password
 <ident prodtype="rhel7" cce="26923-3" />
 <oval id="accounts_password_pam_unix_remember" value="var_password_pam_unix_remember" />
 <ref prodtype="rhel7" nist="IA-5(f),IA-5(1)(e)" disa="200" ossrg="SRG-OS-000077-GPOS-00045" 
-stigid="010240" pcidss="Req-8.2.5" cis="6.3.4" cui="3.5.8" />
+stigid="010240" pcidss="Req-8.2.5" cis="6.3.4" cjis="5.6.2.1" cui="3.5.8" />
```
Comment (shawndwells) [inline shared/xccdf/system/accounts/pam.xml]: Would go a bit more granular, 5.2.1.1.1

ref:
image

**Comment by shawndwells** [comment]
will fix those manually & merge

**Comment by redhatrises** [comment]
Thanks @shawndwells!

**Pull request merged**